src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpSamDAO.scala

```diff
@@ -106,7 +106,9 @@ class HttpSamDAO( implicit val system: ActorSystem, val materializer: Materializ
 
   private def userAuthedRequestToUnit(request: HttpRequest)(implicit userInfo: WithAccessToken): Future[Unit] = {
     userAuthedRequest(request).map { resp =>
-      if(resp.status.isSuccess) Future.successful(())
+      if(resp.status.isSuccess) Future.successful({
+        resp.discardEntityBytes()
+      })
```

Comment:
this should fix errors in logs like
```[WARN] [07/15/2021 17:00:22.965] [FireCloud-Orchestration-API-akka.actor.default-dispatcher-16] [FireCloud-Orchestration-API/Pool(shared->https://sam.dsde-dev.broadinstitute.org:443)] [1 (WaitingForResponseEntitySubscription)]Response entity was not subscribed after 10 seconds. Make sure to read the response `entity` body or call `entity.discardBytes()` on it -- in case you deal with `HttpResponse`, use the shortcut `response.discardEntityBytes()`. PUT /api/group/TCGA-dbGaP-Authorized/member Strict(2 bytes) -> 201 Created Chunked```

Reply:
thank you - we tried to catch all these when migrating from spray->akka but know we missed a few